.gitignore

```diff
@@ -20,6 +20,3 @@ cmd/verify/verify
 
 pkg*
 bin*
-
-# Ignore fake certificates generated for tests
-testdata/fake-certificates*
```

Comment:
These are no longer ignored because we will now use fake certificates that expire in 100 years. This was because CircleCI was upset about using a Go script to create fake test certificates remotely. They'd previously been created in a Bash script, which I stripped to eliminate drift between the Go and Bash cert generation methods.